dask/dataframe/core.py

```diff
@@ -1494,6 +1495,12 @@ def _reduction_agg(self, name, axis=None, skipna=True, split_every=False, out=No
                 result.divisions = (self.columns.min(), self.columns.max())
             return handle_out(out, result)
 
+    def _mode_chunk(df, dropna):
+        if dropna:
+            return Counter(df.dropna().values)
```

Comment:
Can you instead use `df.value_counts(dropna=dropna)`?

Reply:
Thanks for suggesting that.  I wasn't aware of value_counts, but looks like it will simplify things quite a bit.

Reply:
Now things are using value_counts